iModelCore/ECPresentation/Source/Hierarchies/NavNodesCache.cpp

```diff
@@ -1794,7 +1794,7 @@ DataSourceInfo NodesCache::CreateDataSourceInfo(DataSourceIdentifier identifier,
                     hasNodes = stmt->GetValueBoolean(8);
 
                 if (0 != (DataSourceInfo::PART_CustomJson & partsToGet) && !stmt->IsColumnNull(9))
-                    customJson = Json::Value::From(stmt->GetValueText(9));
+                    (BeJsValue)customJson = stmt->GetValueText(9);

```

Comment:
`customJson` is a BeJsDocument and you can't use `=` operator on it.  With `=` operator on BeJsValue you can easily assign primitive values. So I need to cast customJson to BeJsValue and just then I can assign a string.

Reply:
I understand that part, but it's not what I'm asking. `stmt->GetValueText(9)` returns a string and I believe `Json::Value::From` parses JSON from string. I think assigning a string to `BeJsValue` will assign it a string rather than JSON parsed from it, so the behavior changes from what we had before.

Reply:
fixed